### PR TITLE
Add more information about riskiness

### PR DESCRIPTION
**Pull request by beshaya**
Clarifies in the text that each Ibashan gets 3,000 microCovids to spend.
Adds some death rate equivalents to help people decide their budget.

### CODE REVIEW

**Comment by beshaya** [comment]
Related to #319

File: src/posts/paper/2-riskiness.ts [hunk 1]
```diff
@@ -76,17 +76,33 @@ Let’s now explore how to quantify the risk of various activities in terms of m
...
 
+     You may choose to follow a different risk budget. Here is some information that might be helpful in deciding what is right for you:
+     * Among people under 40, the chance of death from COVID is about 0.1-0.2% ([Our World in Data](https://ourworldindata.org/mortality-risk-covid#case-fatality-rate-of-covid-19-by-age))
+     * Among people over 60, this rises to 2-3% and doubles with every 10 years over 70.
+     * A 0.1% chance of death is equivalent to...
+       * Driving 100,000 miles ([IIHS](https://www.iihs.org/topics/fatality-statistics/detail/state-by-state))
+       * Base jumping for 16 years ([Wikipedia](https://en.wikipedia.org/wiki/BASE_jumping#Safety))
```
Comment (blanchardjeremy) [inline src/posts/paper/2-riskiness.ts:101]: Base jumping how often for 16 years?

Comment (beshaya) [reply src/posts/paper/2-riskiness.ts]: Updated to # of jumps.

**Review by blanchardjeremy — APPROVED** [review]
Thumbs up from me but I think someone with a better grasp on the science should review it before merging.

**Comment by catherio** [comment]
pardon the delay but yes these are small enough and simple enough changes that it was fully appropriate not to get 👍  :) :)

**Pull request merged**